rsts/concepts/versioning.rst

```diff
@@ -0,0 +1,45 @@
+.. _divedeep-versioning:
+
+##########
+Versions
+##########
+One of the most important features and reasons for certain design decisions in Flyte is the need for Machine Learning and Data practitioners to experiment.
+When users are experimenting, they usually work in isolation and try multiple iterations. Unlike traditional software, it is necessary for the users to be able to change versions and sometimes conduct multiple experiments concurrently. This may happen when different data scientists are iterating on the same workflow/pipeline concurrently.
```

Comment:
What does ``to be able to change versions`` mean?

Reply:
improved